### DISCUSSION TITLE
While npm run build; The input-h1 class does not exist. If input-h1 is a custom class, make sure it is defined within a @layer directive.

### DISCUSSION THREAD

**Discussion by Bishwas-py** [tailwindlabs/tailwindcss#9364] [Help]
I have created multiple utilities classes and used it in svelte files as styles, but when ever I try to build the app to deploy it on my VPS. I am getting the issue of  
```
The "custom-class" class does not exist.  If "custom-class" is a custom class, make sure it is defined within a `@layer` directive.
```

Help me fix it...

File `app.postcss`:

```CSS
..
@layer utilities {
    .input-h1 {
        @apply text-4xl font-extrabold;
        @apply font-play-fair mx-auto text-center;
        @apply text-gray-800 dark:text-gray-100;
    }
}
..
```

File `form.svelte`

```html
<style>
    h1 {
        @apply input-h1;
    }
</style>
```

**Comment by wongjn** [comment]
The style block in `form.svelte` never "sees" `app.postcss` – each `<style>` block in each component and any CSS files are run in isolation from each other. This means that in `form.svelte`, it never sees `.input-h1` since that is in its own file. [Read more about this in the documentation](https://tailwindcss.com/docs/adding-custom-styles#layers-and-per-component-css).

For your issue, I would suggest using the `input-h1` class directly on the `<h1>` tag.

**Reply by adamwathan** [reply]
> For your issue, I would suggest using the input-h1 class directly on the \<h1\> tag.

110% agree, that's the intended workflow 👍
